src/auth/views/NewPassword.tsx

```diff
@@ -8,19 +8,27 @@ import NewPasswordPage, {
   NewPasswordPageFormData
 } from "../components/NewPasswordPage";
 import { SetPasswordMutation } from "../mutations";
-import { SetPassword } from "../types/SetPassword";
+import {
+  SetPassword,
+  SetPassword_setPassword_errors
+} from "../types/SetPassword";
 import { NewPasswordUrlQueryParams } from "../urls";
 
 const NewPassword: React.FC<RouteComponentProps> = ({ location }) => {
   const navigate = useNavigator();
   const { loginByToken } = useUser();
+  const [errors, setErrors] = React.useState<
```

Comment:
The state here is not needed. `SetPasswordMutation` returns `setPasswordOpts`, which has `data` property, allowing us to access `data.setPassword.errors`, which we can directly convey to `NewPasswordPage` as prop. I'll look like this
```
<NewPasswordPage
  errors={setPasswordOpts.data.?.setPassword.errors || []}
  ...
/>
```

Reply:
You're right. done.

Thank you ;)